packages/maskbook/src/components/shared/InjectedDialog.tsx

```diff
@@ -36,10 +36,12 @@ export type InjectedDialogClassKey =
     | 'dialogActions'
     | 'dialogTitleTypography'
     | 'dialogCloseButton'
+    | 'dialogBackdropRoot'
 export interface InjectedDialogProps extends withClasses<InjectedDialogClassKey>, React.PropsWithChildren<{}> {
     open: boolean
-    onExit?(): void
-    title: React.ReactChild
+    headless?: boolean
```

Comment:
headless actually means "with no UI" and confuse me. After seeing what it did I guess test on truthness of "title" props is enough.